README.md

```diff
@@ -1,77 +1,246 @@
-    <br />
-    <br />
-    <a href="https://henask12.github.io/hello-microverse/">View Demo</a>
-    ·
-    <a href="https://github.com/henask12/hello-microverse/issues">Report Bug</a>
-    ·
-    <a href="https://github.com/henask12/hello-microverse/issues">Request Feature</a>
-  </p>
-</p>
+<a name="readme-top"></a>
+
+<!--
+HOW TO USE:
+This is an example of how you may give instructions on setting up your project locally.
+
+Modify this file to match your project and remove sections that don't apply.
+
+REQUIRED SECTIONS:
+- Table of Contents
+- About the Project
+  - Built With
+  - Live Demo
+- Getting Started
+- Authors
+- Future Features
+- Contributing
+- Show your support
+- Acknowledgements
+- License
+
+OPTIONAL SECTIONS:
+- FAQ
+
+After you're finished please remove all the comments and instructions!
+-->
+
+<div align="center">
+  <!-- You are encouraged to replace this logo with your own! Otherwise you can also remove it. -->
+  <img src="murple_logo.png" alt="logo" width="140"  height="auto" />
+  <br/>
+
+  <h3><b>Hello Microverse Project</b></h3>
+
+</div>
+
 <!-- TABLE OF CONTENTS -->
-Table of Contents
-About the Project
-Built With
-Getting Started
-Installation
-Usage
-Contributing
-License
-Contact
-<!-- ABOUT THE PROJECT -->
-About The Project
-!Hello Microverse
-
-This is a simple website that displays a greeting message to the world. It was created as part of the Microverse curriculum.
-
-Built With
-HTML
-CSS
-JavaScript
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 Hello Microverse <a name="about-project"></a>
+
+> Describe your project in 1 or 2 sentences.
+
+**Hello Microverse** is a...
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+> Describe the tech stack and include only the relevant sections that apply to your project.
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="#">Html</a></li>
+   <li><a href="#">CSS</a></li>
+  </ul>
+</details>
+
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+> Describe between 1-3 key features of the application.
+
+- **simple Hellow world**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+> Add a link to your deployed project.
+
+- [Live Demo Link](https://github.com/henask12/hello-microverse/pulls)
+
```

Comment:
Since your project is not deployed, please let's consider removing the Live demo link to avoid broken links in our readme. 🤗

We can put a text e.g `Live demo is not available` instead 💪